Source/platform/locale.cpp

```diff
@@ -176,12 +176,12 @@ std::vector<std::string> GetLocales()
 
 	CFRelease(languages);
 #else
-	try {
-		std::string locale = std::locale("").name();
-		// strip off any encoding specifier, devX uses UTF8.
-		locales.emplace_back(locale.substr(0, locale.find('.')));
-	} catch (std::runtime_error &e) {
-		LogWarn("Locale detection failed: {}", e.what());
+	const string_view locale = setlocale(0, nullptr);
+	const string_view lang = locale.substr(0, locale.find('.'));
+	if (!locale.empty()) {
+		locales.emplace_back(std::string(lang));
+	} else {
+		LogWarn("Locale detection failed: {}", locale);

```

Comment:
This is logging a string we know is empty.
```suggestion
		LogWarn("Locale detection failed");
```